lib/api/keys.ts

```diff
@@ -17,7 +19,17 @@ export const getKeySalts = () => ({
  * @param {Object} SignedKeyList
  * @return {Object}
  */
-export const createAddressKeyRoute = ({ AddressID, Primary, PrivateKey, SignedKeyList }) => ({
+export const createAddressKeyRoute = ({
```

Comment:
I believe destructuring was used in order to make it more clear what exactly the params were. It's unnecessary now, since we have type for it. Could save some space:

```
export const createAddressKeyRoute = (data: {
    AddressID: string;
    Primary: number;
    PrivateKey: string;
    SignedKeyList: SignedKeyList;
}) => ({
    url: 'keys',
    method: 'post',
    data
});
```